dashboard/app/views/levels/editors/fields/_validation_code.html.haml

```diff
@@ -11,7 +11,11 @@
       = f.label :failure_message_override, 'Failure Message Override'
       %p If specified this error message will be used to replace ALL error messages in the puzzle. Be certain this is what you want before using.
       = f.text_field :failure_message_override, style: 'width: 600px;', placeholder: 'Type Override Failure Message Here'
-  -# Dance and Spritelab are special types of Gamelab levels
+  -# Dance and Poetry are special types of GamelabJr (Sprite Lab) levels
+  - if @level.is_a?(GamelabJr) && !(@level.is_a?(Dancelab) || @level.is_a?(Poetry))
+    %p
+      Sprite Lab validation can utilize a number of <a href="https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/p5lab/spritelab/commands/criterionCommands.js">criterion commands</a>. Available feedback strings are located in <a href="https://github.com/code-dot-org/code-dot-org/blob/staging/apps/i18n/spritelab/en_us.json">spritelab/en_us.json</a>. Additional criterion functions and feedback strings need to be added by an engineer.
```

Comment:
nit: can we wrap this?  We appear to wrap plain text later on in this file..

Reply:
Good call. I was following the convention I saw four lines down because I had limited knowledge of how whitespace works in .haml files. I can propose a formatting fix for this whole file after checking that the wrapping doesn't cause any issues.